Decred Wallet/Features/Overview/OverviewViewController.swift

```diff
@@ -101,12 +101,20 @@ class OverviewViewController: UIViewController {
         super.viewWillAppear(animated)
         self.navigationController?.navigationBar.isHidden = true
         self.checkWhetherToPromptForSeedBackup()
+        self.updateMultiWalletBalance()
+        self.updateRecentActivity()
```

Comment:
Call `self.refreshRecentActivityAndUpdateBalance()` instead. This would show the refresh control spinner while performing these updates.
Also, `self.checkWhetherToPromptForSeedBackup()` should be the last function call, because that function is implemented much lower in this file.

As a rule of thumb, call functions in the same order that they are implemented. Or put more correctly, implement functions in the order that they are called. Makes it easier to find functions by looking just around where they are called.

Reply:
resolved in `698f737 `